Cargo.lock

```diff
@@ -7254,7 +7254,7 @@ checksum = "f271a476bbaa9d2139e1e1a5beb869c6119e805a0b67ad2b2857e4a8785b111a"
 dependencies = [
  "prettyplease 0.2.15",
  "proc-macro2",
- "prost-build 0.13.4",
+ "prost-build 0.13.1",
```

Comment:
Downgrade?

Reply:
prost-build 0.13.4 actually depends on prost 0.13.4

It seems the `patch` somehow bypassed this 🤔

Anyway, I decided to bump to 0.13.4